src/Codata/Sized/Conat/Properties.agda

```diff
@@ -8,13 +8,13 @@
 
 module Codata.Sized.Conat.Properties where
 
-open import Size
+open import Size using (Size)
 open import Data.Nat.Base using (ℕ; zero; suc)
-open import Codata.Sized.Thunk
+open import Codata.Sized.Thunk using (Thunk; Thunk^R; force)
 open import Codata.Sized.Conat
-open import Codata.Sized.Conat.Bisimilarity
+open import Codata.Sized.Conat.Bisimilarity using (_⊢_≈_; refl; zero; suc)
 open import Function.Base using (_∋_)
-open import Relation.Nullary
+open import Relation.Nullary using (¬_; yes; no)
```

Comment:
Pretty sure you can lower `¬_` to come from `Relation.Nullary.Negation` and then `yes; no` come from `Relation.Nullary.Decidable`.

Reply:
Indeed, in each case, from `*.Core`...